examples/web-server-da/orchestrator.rs

```diff
@@ -0,0 +1,28 @@
+pub mod types;
+
+use clap::Parser;
+use hotshot::demos::sdemo::SDemoTypes;
+use tracing::instrument;
+use types::ThisMembership;
+
+use crate::infra::OrchestratorArgs;
+use crate::infra_da::run_orchestrator_da;
+use crate::types::{DANetwork, NodeImpl, QuorumNetwork};
+
+#[path = "../infra/mod.rs"]
+pub mod infra;
+#[path = "../infra/modDA.rs"]
+pub mod infra_da;
+
+#[cfg_attr(
+    feature = "tokio-executor",
+    tokio::main(flavor = "multi_thread", worker_threads = 2)
+)]
+#[cfg_attr(feature = "async-std-executor", async_std::main)]
+#[instrument]
+async fn main() {
```

Comment:
nit (maybe this is done elsewhere?): might be worth running `setup_logging` and `seup_backtrace`.